packages/systemd-257/org.freedesktop.login1.toml

```diff
@@ -0,0 +1,515 @@
+# Generated programatically, serde/toml doesn't allow to force the "compact"
+# format on serialization:
+# [user.root]
+# rules = []
+
+[[user.root.rules]]
+own = "org.freedesktop.login1"
+allow = true
+
+[[user.root.rules]]
+send_destination = "org.freedesktop.login1"
+allow = true
+
+[[user.root.rules]]
+receive_sender = "org.freedesktop.login1"
+allow = true
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+allow = false
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+send_interface = "org.freedesktop.DBus.Introspectable"
+allow = true
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+send_interface = "org.freedesktop.DBus.Peer"
+allow = true
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+send_interface = "org.freedesktop.DBus.Properties"
+send_member = "Get"
+allow = true
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+send_interface = "org.freedesktop.DBus.Properties"
+send_member = "GetAll"
+allow = true
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+send_interface = "org.freedesktop.login1.Manager"
+send_member = "GetSession"
+allow = true
+
+[[default.rules]]
+send_destination = "org.freedesktop.login1"
+send_interface = "org.freedesktop.login1.Manager"
+send_member = "GetSessionByPID"
+allow = true
```

Comment:
Not a blocker but these rules would be a lot easier to read if we allowed a list of methods per (destination, interface).
```
[[default.rules]]
send_destination = "org.freedesktop.login1"
send_interface = "org.freedesktop.login1.Manager"
send_members = [
  "GetSession",
  "GetSessionByPID",
]
allow = true
```

That would significantly reduce the boilerplate in these files.